src/Inciter/AMR/Refinement_State.h

```diff
@@ -0,0 +1,167 @@
+//
+// Created by Robert Francis Bird on 10/27/16.
+//
+
+#ifndef TETAMR_AMR_DATA_H
+#define TETAMR_AMR_DATA_H
+
+#include <vector>
+#include <cstddef>
+
+#include "types.h"
+
+namespace AMR {
+    // Types
+    using real_t = tk::real;
+
+    const size_t DEFUALT_CHILD_NUMBER = 0; // Used to mark "doesn't have children"
+    const size_t DEFAULT_NUM_CHILDREN = 0; // Default to no children
+
+    // TODO: Populate this enum with the available refinement cases
+    enum Refinement_Case { initial_grid, one_to_two, one_to_four, one_to_eight,
```

Comment:
As we already discussed, it would probably be better to convert these enums to type-safe ones.